### DISCUSSION TITLE
Google Analytics Support

### DISCUSSION THREAD

**Discussion by rendybjunior** [ragapp/ragapp#254] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I am using ragapp for internal HR chatbot and I would like to measure the usage to estimate the impact of the chatbot.

**Describe the solution you'd like**
Google analytics option,  an environment variable to put tracking id

**Describe alternatives you've considered**
- GTM would be another option
- database logging of usage also an option

**Additional context**
No

**Comment by rendybjunior** [comment]
I just found out that we have tracking snippet, can we use this for GA? By changing the script using tagging script from GA.
https://github.com/ragapp/ragapp/tree/main/deployments/single#add-tracking-script

**Reply by marcusschiesser** [reply, marked as answer]
yes, exactly, you can use this for adding GA
